realm/realm-library/src/main/java/io/realm/RealmObjectSchema.java

```diff
@@ -525,7 +592,7 @@ Table getTable() {
     }
 
     static final Map<Class<?>, FieldMetaData> getSupportedSimpleFields() {
```

Comment:
The function name is misleading, it is just for lists. We could directly access the `SUPPORTED_LIST_SIMPLE_FIELDS ` as you do with `SUPPORTED_DICTIONARY_SIMPLE_FIELDS`.